crates/release_plz/src/args/release.rs

```diff
@@ -29,6 +29,9 @@ pub struct Release {
     /// Publish GitHub release for the created git tag.
     #[clap(long)]
     pub git_release: bool,
+    /// Flags to pass to `cargo publish`
+    #[clap(long)]
+    pub publish_flags: Option<String>,
```

Comment:
```suggestion
    #[clap(long, value_delimiter = ' ')]
    pub publish_flags: Vec<String>,
```
You can let clap do the parsing :)

Reply:
I did not know that! Thanks!

Reply:
Fixed!